using-mobile-extensions/adobe-media-analytics/migration.md

```diff
@@ -0,0 +1,947 @@
+# Migrating to AEPMedia reference
+
+This document is a reference comparison of AEPMedia \(3.x\) APIs against their equivalent ACPMedia \(2.x\) APIs.
+
+The AEPMedia extension is implemented purely in Swift and is compatible with the AEPCore swift SDK. To ensure a smooth transition from the ACPMedia SDK, there are no major changes on the API names or definition. For more details, follow the migration guide below for your Swift or Objective-C mobile application.  If explanation beyond showing API differences is necessary, it will be captured as an info hint within that API's section.
+
+## Public classes
```

Comment:
It feels a little weird to say "Public classes" here when, by definition, these docs are public. I'd go with "AEPMedia classes", or something like that instead.